scripts/check-env.sh

```diff
@@ -37,6 +37,11 @@ else
    pip install tox
 fi
 
+echo "===> Installing couchdb"
+sudo apt update
```

Comment:
I don't think we can use `sudo` there.

Reply:
I'm not sure how we can install `couchdb` on the azure instance.

Reply:
We can pick the a popular linux, e.g., Ubuntu 18.04.

Reply:
For the Azure VM @yeasy ? if so, it already uses `Ubuntu 18.04 image`.